content/concepts/concepts.md

```diff
@@ -0,0 +1,129 @@
+```
+  module.exports = config; 
+
+  var config = {
+    entry: './path/to/my/entry/file.js',
+    output: {
+      filename: 'my-first-webpack.bundle.js',
+      path: __dirname + 'dist/'
+    }
+  }
+
+```
+
+In the example above, through the `output.filename` and `output.path` properties we are describing to webpack the name of our bundle, and where we want it to be emitted to.
+
+T> You may see the term **emitted** or **emit** used throughout our documentation and [plugin API](../api/plugins). This is a fancy term for "produced or discharged". 
+
+The `output` property has [many more configurable features](../api/configuration), but lets spend some time understanding some of the most common use cases for the `output` property. 
+
+[**Learn more!**](./output)
+
+### Loaders
+The goal is to have all of your assets in your project to be **webpack's** conscern and not the browser. (This doesn't mean that they all have to be bundled together). webpack treats [every file (.css, .html, scss, jpg, etc.) as a module](./everything-is-a-module). However, webpack **only understands javascript by default**.
+
+**Loaders tell webpack _how to treat these files as modules_ as they are added to your dependency graph.**
+
+At a high level, they have two purposes in your webpack config. 
+1. Identify what files should be transformed by a certain loader. (`test` property)
+2. Transform that file so that it can be added to your dependency graph (and eventually your bundle). (`loader` property)
+
+**webpack.config.js**
+```
+  module.exports = config; 
+
+  var config = {
+    entry: './path/to/my/entry/file.js',
+    output: {
+      filename: 'my-first-webpack.bundle.js',
+      path: __dirname + 'dist/'
+    },
+    module: {
+      loaders: [
+        {test: /.jsx?$/, loader: 'babel-loader'} 
+      ]
+    }
+  };
+```
+
+In the configuration above we have defined our loader with its two required properties: `test`, and `loader`. It tells webpack's compiler the following: 
+
+```"Hey webpack compiler, when you come across a file with extension '.js' or '.jsx' inside of a require() statement, use the babel-loader to transform it before you bundle it together".```
+
+W> It is important to remember when defining loaders in your webpack config, you are defining them under `module.loaders`, and not `loaders`.
+
+There are more specific properties to define on loaders that we haven't yet covered. 
+
+[**Learn more!**]('./loaders')
+
+### Plugins
+Since Loaders only execute transforms on a per-file basis, Plugins are most commonly used (but not limited to) performing actions and custom functionality on "compilations" or "chunks" of your bundled modules [(and so much more)](./plugins). The webpack Plugin system is [extremely and powerful and customizable](../api/plugins). 
+
+Because a plugin is an ES5 JavaScript class, using it is as trivial as `require()`ing it into your configuration and passing a **`new` instance** of that class to the `plugins` array. 
```

Comment:
"In order to use a plugin, you just need to `require()` it and add it to the `plugins` array. Since most plugins are customizable via options, you need to create an instance of it by calling it with `new`"